run-tests.pl

```diff
@@ -185,6 +201,36 @@ sub usage
 Getopt::Long::Configure("no_passthrough");
 GetOptions($HS_FACTORY->get_options()) or usage(1);
 
+# Check if both options have been set
+if (length($BLACKLIST_FILE) and length($WHITELIST_FILE)) {
```

Comment:
you can just use truthiness here:

```suggestion
if( $BLACKLIST_FILE and $WHITELIST_FILE ) {
```

likewise below